app/scripts/components/common/map/controls/aoi/index.tsx

```diff
@@ -9,18 +10,36 @@ import { computeDrawStyles } from './style';
 
 type DrawControlProps = MapboxDraw.DrawOptions;
 
+// Overriding the dragMove and dragFeature methods of the 
+// 'simple_select' and the 'direct_select' modes to avoid
+// accidentally dragging the selected or hand-drawn AOIs 
+const customSimpleSelect = {
+  ...MapboxDraw.modes.simple_select,
+  dragMove() { return; }
+};
+
+const customDirectSelect = {
+  ...MapboxDraw.modes.direct_select,
+  dragFeature() { return; },
+};
+
 export default function DrawControl(props: DrawControlProps) {
   const theme = useTheme();
   const control = useRef<MapboxDraw>();
   const aoisFeatures = useAtomValue(aoisFeaturesAtom);
-
   const { onUpdate, onDelete, onSelectionChange, onDrawModeChange } = useAois();
 
   useControl<MapboxDraw>(
     () => {
       control.current = new MapboxDraw({
         displayControlsDefault: false,
         styles: computeDrawStyles(theme),
+        modes: {
+          ...MapboxDraw.modes,
+          simple_static: StaticMode,
+          simple_select: customSimpleSelect,
```

Comment:
At this point, what is the difference between `simple_static` and `simple_select` mode?

Reply:
I should have really named this only `static` instead of `simple_static`, but it's basically two different modes. `simple_static` disables all user interactions (including dragging, and prevents vertices from showing up per polygon) because that's the behavior we want to try out for the presets. The `simple_select` mode is almost the same as before, aside from the dragging of the feature as a whole being disabled (but vertices are still shown and can be interacted with).

So `simple_static` is only for the presets at the moment, while the `simple_select` is for uploaded assets or hand-drawn features.

Reply:
https://github.com/mapbox/mapbox-gl-draw-static-mode